test/aws/osml/model_runner/tile_worker/test_tile_worker_utils.py

```diff
@@ -85,10 +100,13 @@ def test_process_tiles(
         )
         region_request_item = RegionRequestItem.from_region_request(mock_region_request)
 
-        ds, sensor_model = load_gdal_dataset("./test/data/small.ntf")
-        mock_elevation_model = None
-        work_queue, tile_worker_list = setup_tile_workers(mock_region_request, sensor_model, mock_elevation_model)
+        # Mock the GDAL dataset loading
```

Comment:
NIT: This isn't mocking anything. It looks like this loads an actual test datasset and sensor model. Not sure creating this get_dataset_and_camera wrapper around load_gdal_dataset is actually getting you anything and it actually makes it harder to create tests with different images.

Reply:
I will try and go to a mocking approach and update the PR - ty!